settings.gradle.kts

```diff
@@ -52,7 +52,7 @@ dependencyResolutionManagement {
             version("eddsa-version", "0.3.0")
             version("grpc-version", "1.39.0")
             version("guava-version", "31.1-jre")
-            version("hapi-version", "0.30.0-SNAPSHOT")
+            version("hapi-version", "0.30.0-alias-SNAPSHOT")
```

Comment:
This will need to be changed prior to checkin.

Reply:
not sure what to do about this.  I believe the build will fail if I use `0.30.0-SNAPSHOT`.  The first version of `0.30.0-SNAPSHOT` did not have some of the updates to the protobuf that I needed.  Neeha created a new version that seems to work for me in my local but I had to delete the gradle cache to get the new version.  I'm not sure how to get this done on the build machine.

Reply:
If we can't fix it in this then just get a PR ready after this that updates this line. After this PR goes in, rebase and get that one in.
Or open a new PR here that we all go sign off on.